apps/playlist/apis/playlist/v0alpha1/playlist_spec_gen.go

```diff
@@ -0,0 +1,36 @@
+package v0alpha1
+
+// Defines values for PlaylistItemType.
```

Comment:
not related to this PR specifically, but would be nice if this stayed an enum value

Reply:
I'm not sure what we mean here, do you mean adding the additional `// +enum` comment to `PlaylistItemType`? We can try to get that working with the [deepmap/oapi-codegen](https://github.com/deepmap/oapi-codegen) we currently use to do the go type codegen.